sentry_sdk/scope.py

```diff
@@ -1202,40 +1425,170 @@ def update_from_kwargs(
         if fingerprint is not None:
             self._fingerprint = fingerprint
 
-    def __copy__(self):
-        # type: () -> Scope
-        rv = object.__new__(self.__class__)  # type: Scope
-
-        rv._level = self._level
-        rv._name = self._name
-        rv._fingerprint = self._fingerprint
-        rv._transaction = self._transaction
-        rv._transaction_info = dict(self._transaction_info)
-        rv._user = self._user
-
-        rv._tags = dict(self._tags)
-        rv._contexts = dict(self._contexts)
-        rv._extras = dict(self._extras)
-
-        rv._breadcrumbs = copy(self._breadcrumbs)
-        rv._event_processors = list(self._event_processors)
-        rv._error_processors = list(self._error_processors)
-        rv._propagation_context = self._propagation_context
-
-        rv._should_capture = self._should_capture
-        rv._span = self._span
-        rv._session = self._session
-        rv._force_auto_session_tracking = self._force_auto_session_tracking
-        rv._attachments = list(self._attachments)
-
-        rv._profile = self._profile
-
-        return rv
-
     def __repr__(self):
         # type: () -> str
         return "<%s id=%s name=%s>" % (
             self.__class__.__name__,
             hex(id(self)),
             self._name,
         )
+
+
+@contextmanager
+def new_scope(scope=None):
```

Comment:
please remove the scope argument here like you did in `isolation_scope` below, since there's a new `use_scope` method for using an explicitly passed in scope.

Reply:
Hrm. Thought I did this. Removed now.